app/helpers/accounts_helper.rb

```diff
@@ -21,4 +21,8 @@ def available_accounts_array
   def available_accounts_options
     options_for_select(available_accounts_array, @order_detail.account_id)
   end
+
+  def account_owner_name(account)
+    account.owner_user.try(:name) || ""
```

Comment:
Would we be better off if we changed `Account#owner_user` to return some sort of null object?